db.js

```diff
@@ -0,0 +1,118 @@
+require("dotenv").config()
+const { Client } = require("pg");
+
+const client = new Client({
+  user: process.env.DB_USER,
+  host: process.env.DB_HOST,
+  database: process.env.DB_NAME,
+  password: process.env.DB_PASSWORD,
+  port: Number(process.env.DB_USER),
+});
+
+client
+  .connect()
+  .then(() => console.log("Connected to database"))
+  .catch((err) => console.error("Error connecting to database: ", err.stack));
+
+async function closeConnection() {
+  client
+    .end()
+    .then(() => console.log("Connection ended"))
+    .catch((err) => console.error("Error closing connection: ", err.stack));
+}
+
+async function fetchTodos() {
+  try {
+    res = await client.query("SELECT * FROM todo");
+    if (res.rowCount === 0) {
+      console.log("No tasks found");
+      return;
+    } else {
+      res.rows.forEach((row) =>
+        console.log(`${row.id}. ${row.task} [${row.done ? "x" : ""}]`)
```

Comment:
You can map the response and transform it into the format you want. So you can do something like 
```javascript
res.rows.map(row => `${row.id}. ${row.task} [${row.done ? "x" : ""}]`)
```

Reply:
It's a cli project i don't know if it would be shown on the terminal without the console log but i go try

Reply:
you can do that, assign it to a variable and then print it out

Reply:
@bra-alex without forEach it would print out an array containing the tasks instead of printing the tasks line by line